docs/eep.schema

```diff
@@ -105,6 +105,10 @@
                                     "description": "Pull resistor setting for this gpio",
                                     "enum": [ "default", "up", "down", "none" ]
                                 }
+                                "comment": {
```

Comment:
There is a comma missing after the "pull"-property block before the "comment".

                                },
                                "comment": {

Reply:
done